data/board.ts

```diff
@@ -0,0 +1,83 @@
+export type SocialMap = {
+  mastodon?: string
+}
+
+export type BoardMember = {
+  name: string
+  position: string
+  title?: string
+  otherTitle?: string
+  socials?: SocialMap
+  slug?: string
+  avatar?: string
+}
+
+export const BOARD_POSITION_DIRECTOR = "Board Director";
```

Comment:
Those are not localizable right now I think?

Maybe we should use a symbol/constant for those, and use a translatable string in the page where they are displayed?

It looks cleaner this way, if someday we want to translate this page.

Reply:
> Those are not localizable right now I think?

Yeah, not localizable.

> Maybe we should use a symbol/constant for those, and use a translatable string in the page where they are displayed?

Yep, makes sense. Will update